app/src/main/java/com/github/onlynotesswent/ui/user/PublicProfile.kt

```diff
@@ -661,3 +707,24 @@ fun NotificationButton(
             else Color.Unspecified)
   }
 }
+
+@Composable
+fun SendMessageButton(
+    userViewModel: UserViewModel,
+    notificationViewModel: NotificationViewModel,
+    onSendMessageClick: () -> Unit
+) {
+  userViewModel.currentUser.collectAsState().value?.let { user ->
```

Comment:
what is the purpose of this?

Reply:
Not any purpose I taught it will be good to refresh the notifications but in this case it serves no purpose so I will remove it